Authenticator-related updates (#620)
usnistgov/FIPS201: _FIPS201/authentication.md

 If the symmetric card authentication key is present, it **SHALL** be used for PIV cardholder authentication using the following
steps:

> - The CHUID or another data element[^expired2] is read from the card. The signature of the CHUID or another data element is verified to ensure that the card has not expired and that the card comes from a trusted source.

Reviewer: ```suggestion
- The CHUID or another data element[^expired2] is read from the card and is checked to ensure that the card has not expired.
```

```suggestion
- The CHUID or another data element[^expired2] is read from the card. The signature of the CHUID or another data element is verified to ensure that the card has not expired and that the card comes from a trusted source.
```

The following bullets talks about signature checks and 'trusted source' .  That is why I removed it here.

[Table 6-2. Applicable PIV Authentication Mechanisms for Logical Access](authentication.md#table-6-2){:name="table-6-2"}
{:latex-ignore="true"}

|Required Authenticator Assurance Level|Local Authentication|Remote/Network Authentication|
|---|---|---|
|AAL1|PKI-CAK|PKI-CAK|
|AAL2|BIO||
|AAL3|BIO-A, OCC-AUTH, PKI-AUTH|PKI-AUTH|
{:latex-table="6-2" latex-caption="Applicable PIV Authentication Mechanisms for Logical Access"}

> In the above table, local authentication refers to authentication to a resource co-located with the cardholder, while remote/network authentication is digital authentication of the cardholder to a system over a network.

Author: ```suggestion
In the above table, local authentication refers to authentication to a resource co-located with the cardholder, while remote/network authentication is authentication of the cardholder to a system over a network.
```
